modules/ROOT/pages/bare-metal.adoc

```diff
@@ -0,0 +1,53 @@
+:experimental:
```

Comment:
What do these `experimental` markers mean? Is it e.g. some notice to say that FCOS is experimental? I think we can drop those now. :)